daffodil-test/src/test/resources/org/apache/daffodil/section00/general/general.tdml

```diff
@@ -217,4 +217,32 @@
         <tdml:error>in no namespace</tdml:error>
     </tdml:errors>
   </tdml:parserTestCase>
+
+  <tdml:defineSchema name="nameAndRef">
+    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd" />
+    <dfdl:format ref="ex:GeneralFormat" />
+    <xs:element name="root1">
+      <xs:complexType>
+        <xs:sequence>
+          <xs:element name="nameAndRef" type="xs:string" ref="ex:root1" dfdl:lengthKind="delimited" />
```

Comment:
Just noticed this belatedly.  The `ref="ex:root1"` should be `ref="ex:root2"` in order to avoid having two errors mixed into the same line.